test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -59,13 +60,15 @@ template <class T> class HttpFilterTestBase : public T {
                                    "ext_authz_prefix", bootstrap_));
     client_ = new Filters::Common::ExtAuthz::MockClient();
     filter_ = std::make_unique<Filter>(config_, Filters::Common::ExtAuthz::ClientPtr{client_});
-    filter_->setDecoderFilterCallbacks(filter_callbacks_);
+    filter_->setDecoderFilterCallbacks(decoder_filter_callbacks_);
```

Comment:
Similar to the network filter test, we can create a time system and assign it to `decoder_filter_callbacks_->dispatcher_.time_system_`.